packages/core/components/DropZone/index.tsx

```diff
@@ -175,6 +175,7 @@ function DropZoneEdit({ zone, style }: DropZoneProps) {
                 const defaultedProps = {
                   ...config.components[item.type]?.defaultProps,
                   ...item.props,
+                  puckCtx: { renderDropZone: DropZone },
```

Comment:
I find `puckCtx` a bit awkward, and think `ctx` (aka "context") is a bit of a misnomer.

What about simply `puck`?

```suggestion
                  puck: { renderDropZone: DropZone },
```